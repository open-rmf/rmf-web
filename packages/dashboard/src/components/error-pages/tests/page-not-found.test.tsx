import React from 'react';
import { render } from '@testing-library/react';
import NotFoundPage from '../page-not-found';
import { BrowserRouter } from 'react-router-dom';

describe('PageNotFound', () => {
  test('renders correctly', () => {
    const root = render(
      <BrowserRouter>
        <NotFoundPage />
      </BrowserRouter>,
    );
    expect(root.queryByAltText('404 Not Found')).toBeTruthy();
    expect(root.queryByText('Are you lost?')).toBeTruthy();
    root.unmount();
  });
});
