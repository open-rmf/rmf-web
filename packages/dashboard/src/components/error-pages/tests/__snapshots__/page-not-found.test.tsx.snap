// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound renders correctly 1`] = `"<div class=\\"makeStyles-div-1\\"><img class=\\"makeStyles-img-2\\" src=\\"assets/aron-visuals-3jBU9TbKW7o-unsplash.jpg\\" alt=\\"404 Not Found\\"><h2 class=\\"MuiTypography-root makeStyles-text-3 MuiTypography-h1 MuiTypography-gutterBottom\\">Are you lost? <a href=\\"/login\\">Go to Login </a></h2><p class=\\"MuiTypography-root makeStyles-author-4 MuiTypography-body1\\">Photo by Aron Visuals on Unsplash</p></div>"`;
